### PR TITLE
Add pillars colours to article

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?

Brings actual pillar support to the article body:

- Content label
- Byline
- Sharing icons
- Body links

## Why?

Visual party 🎉

### CODE REVIEW

File: frontend/components/ArticleBody.tsx [hunk 1]
```diff
@@ -68,6 +69,18 @@ const wrapper = css`
         }
     }
 `;
+const pillarColours = pillarMap(
```
Comment (nicl) [inline frontend/components/ArticleBody.tsx]: Both this and below feel like pretty generic behaviour. Happy to keep them here if it's the only use, but would be good to move them into the pillar package at some point.

File: frontend/components/ArticleBody.tsx [hunk 2]
```diff
@@ -242,22 +252,23 @@ const shareIcon = (colour: string) => css`
...
-
-    :hover {
-        background-color: ${colour};
-        border-color: ${colour};
-        fill: white;
-    }
 `;
-
-const ageWarning = (colour: string) => css`
+const pillarHover = pillarMap(
```
Comment (nicl) [inline frontend/components/ArticleBody.tsx]: Is this used anywhere?

**Comment by SiAdcock** [comment]
In @AWare's absence, I'm going to pick this up today.

File: frontend/components/ArticleBody.tsx (hunk 1 above)
Comment (SiAdcock) [reply frontend/components/ArticleBody.tsx]: I like the idea of extracting these functions, however I don't like making `css` a dependency of the pillars module. Maybe we can move these functions into a new module called `lib/pillar-style-helpers` or something like that? We could move `pillarMap` there too.

File: frontend/lib/parse-capi/index.ts [hunk 3]
```diff
@@ -177,15 +178,11 @@ export const extractArticleMeta = (data: {}): CAPIType => {
         sectionName: getNonEmptyString(data, 'config.page.section'),
         pageId: getNonEmptyString(data, 'config.page.pageId'),
         sharingUrls: getSharingUrls(data),
+        pillar:
+            findPillar(getNonEmptyString(data, 'config.page.sectionName')) ||
+            'news',
```
Comment (SiAdcock) [inline frontend/lib/parse-capi/index.ts]: `sectionName != pillar`

For instance, [this article's](https://www.theguardian.com/fashion/2018/sep/27/which-female-celebrity-embodies-the-challenges-facing-women-today) `sectionName` is "Fashion"

File: frontend/components/ArticleBody.tsx (hunk 1 above)
Comment (AWare) [reply frontend/components/ArticleBody.tsx]: Yeah, I put them in `pillars.ts` to start with, I think this forms part of the more general "where does css live" question too.

**Comment by AWare** [comment]
I'll add the nav changes to another PR.

**Pull request merged**